docs/platform/dedicated-cpu/getting-started-with-dedicated-cpu/index.md

```diff
@@ -35,29 +37,35 @@ For more information on Dedicated CPU use cases, see our [Dedicated CPU Use Case
   {{< /note >}}
 
-1. Choose the region where you would like your Linode to reside. If you’re not sure which to select, see our [How to Choose a Data Center guide](/docs/platform/how-to-choose-a-data-center/). You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the route between you and a data center in each specific region.
+1. Choose the region where you would like your Linode to reside. If you’re not sure which to select, see our [How to Choose a Data Center guide](/docs/platform/how-to-choose-a-data-center/). You can also generate [MTR reports](/docs/networking/diagnostics/diagnosing-network-issues-with-mtr/) for a deeper look at the network route between you and each of our data centers.
 
-2. At the top of the **Linode Plan** section, click on *Dedicated CPU* and select the Dedicated CPU plan you would like to use.
+1. At the top of the **Linode Plan** section, click on the **Dedicated CPU** tab and select the Dedicated CPU plan you would like to use.
 
-3. Enter a label for your new Linode.
+1. Enter a label for your new Linode under the **Linode Label** field.
 
-4. Enter a strong root password for your Linode in the **Root Password** field. This password must be at least six characters long and contain characters from at least two of the following categories:
+1. Enter a strong root password for your Linode in the **Root Password** field. This password must be at least six characters long and contain characters from at least two of the following categories:
 
-  - lowercase and uppercase case letters
-  - numbers
-  - punctuation characters
+    - lowercase letters
+    - uppercase letters
+    - numbers
+    - punctuation characters
+
+    {{< note >}}
+You will not be prompted to enter a root password if you are cloning another Linode or restoring from the Linode Backups service.
+{{< /note >}}
 
 1. Optionally, add an [SSH key](/docs/security/authentication/use-public-key-authentication-with-ssh/#upload-your-ssh-key-to-the-cloud-manager), [Backups](/docs/platform/disk-images/linode-backup-service/), or a [Private IP address](/docs/platform/manager/remote-access/#adding-private-ip-addresses).
 
-1. Click the **Create** button to complete the creation process for your Dedicated CPU Linode, and to be redirected to it's overview page.
-{{< note >}}
-  See our [Getting Started Guide](https://www.linode.com/docs/getting-started/) for more information.
-{{< /note >}}
+1. Click the **Create** button when you have finished completing this form. You will be redirected to the overview page for your new Linode. This page will show a progress bar which will indicate when the Linode has been provisioned and is ready for use.
```

Comment:
Adding some detail about how a Linode won't necessarily be ready once the Create button is clicked, but that the progress will be indicated